internal/controller/vrg_volrep.go

```diff
@@ -158,7 +158,7 @@ func (v *VRGInstance) reconcileVRAsSecondary(pvc *corev1.PersistentVolumeClaim,
 
 	pvcNamespacedName := types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}
 
-	requeueResult, ready, err := v.processVRAsSecondary(pvcNamespacedName, log)
+	requeueResult, ready, err := v.processVRAsSecondary(pvcNamespacedName, pvc, log)
```

Comment:
Doesn't this contradict the intent of the PR? We are passing the PVC name and namespace to create the VR, but we can't use `pvcNamespacedName` to locate the PVC. I understand your intent, but this approach will face the same issue you are trying to safeguard against.

Reply:
@BenamarMk I think today it is the same i.e VR name is the same as the PVC name and should remain so for backward compat. But, in the future the VR name would be a group name and the first argument would change to something other than the pvc name/ns.

The intent is to reuse `processVRAs*` to process a VR or a VRGroup, given a PVC.

Reply:
Agreed. Setting the groundwork in this PR is the first step.